docs/conf.py

```diff
@@ -0,0 +1,60 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# TODO: Use with autodoc
+# from django.conf import settings
+
+# settings.configure()
+
+
+# -- Project information -----------------------------------------------------
+
+project = "xocto"
+copyright = "2023, Kraken Tech"
+author = "Kraken Tech"
+
+# The full version, including alpha/beta/rc tags
+release = "2.2.1"
```

Comment:
can we reference this from setup.py, or move this to a shared file, so that we don't need to update this all around the codebase when we update a version?